packages/scripts/package.json

```diff
@@ -60,6 +61,7 @@
 		"stylelint": "^9.10.1",
 		"stylelint-config-wordpress": "^13.1.0",
 		"thread-loader": "^2.1.2",
+		"url-loader": "^3.0.0",
```

Comment:
This change isn't reflected in the `package-lock.json`.

Reply:
I don't see a package-lock.json file in the scripts directory

Reply:
The one in the root folder only exists and it should reflect the changes applied 😄

Reply:
:+1:

Reply:
:+1: